### DISCUSSION TITLE
so fnkey.pem where is it and how do i get it. (please help me.)

### DISCUSSION THREAD

**Discussion by MrButterfingersss** [RinLovesYou/Flipnote-Signer#13] [Q&A]
well, the title says most of it but I need to know how to because well I don't know maybe Cuz maybe it will help other's or something but i need to know. I can't open my animations on my DSi without it they just won't open please help me I'm so confused. if you can tell me please do. I am open for discussion for obvious reasons.

**Comment by DatGamerYoloYT** [comment]
do a google search
